Libraries/CMSIS/Device/Maxim/MAX32657/Include/uart_regs.h

```diff
@@ -0,0 +1,395 @@
+#define MXC_F_UART_STATUS_RX_EM                        ((uint32_t)(0x1UL << MXC_F_UART_STATUS_RX_EM_POS)) /**< STATUS_RX_EM Mask */
+
+#define MXC_F_UART_STATUS_RX_FULL_POS                  5 /**< STATUS_RX_FULL Position */
+#define MXC_F_UART_STATUS_RX_FULL                      ((uint32_t)(0x1UL << MXC_F_UART_STATUS_RX_FULL_POS)) /**< STATUS_RX_FULL Mask */
+
+#define MXC_F_UART_STATUS_TX_EM_POS                    6 /**< STATUS_TX_EM Position */
+#define MXC_F_UART_STATUS_TX_EM                        ((uint32_t)(0x1UL << MXC_F_UART_STATUS_TX_EM_POS)) /**< STATUS_TX_EM Mask */
+
+#define MXC_F_UART_STATUS_TX_FULL_POS                  7 /**< STATUS_TX_FULL Position */
+#define MXC_F_UART_STATUS_TX_FULL                      ((uint32_t)(0x1UL << MXC_F_UART_STATUS_TX_FULL_POS)) /**< STATUS_TX_FULL Mask */
+
+#define MXC_F_UART_STATUS_RX_LVL_POS                   8 /**< STATUS_RX_LVL Position */
+#define MXC_F_UART_STATUS_RX_LVL                       ((uint32_t)(0xFUL << MXC_F_UART_STATUS_RX_LVL_POS)) /**< STATUS_RX_LVL Mask */
+
+#define MXC_F_UART_STATUS_TX_LVL_POS                   12 /**< STATUS_TX_LVL Position */
+#define MXC_F_UART_STATUS_TX_LVL                       ((uint32_t)(0xFUL << MXC_F_UART_STATUS_TX_LVL_POS)) /**< STATUS_TX_LVL Mask */
+
+/**@} end of group UART_STATUS_Register */
+
+/**
+ * @ingroup  uart_registers
+ * @defgroup UART_INTEN UART_INTEN
+ * @brief    Interrupt Enable control register
+ * @{
+ */
+#define MXC_F_UART_INTEN_RX_FERR_POS                   0 /**< INTEN_RX_FERR Position */
+#define MXC_F_UART_INTEN_RX_FERR                       ((uint32_t)(0x1UL << MXC_F_UART_INTEN_RX_FERR_POS)) /**< INTEN_RX_FERR Mask */
+
+#define MXC_F_UART_INTEN_RX_PAR_POS                    1 /**< INTEN_RX_PAR Position */
+#define MXC_F_UART_INTEN_RX_PAR                        ((uint32_t)(0x1UL << MXC_F_UART_INTEN_RX_PAR_POS)) /**< INTEN_RX_PAR Mask */
+
+#define MXC_F_UART_INTEN_CTS_EV_POS                    2 /**< INTEN_CTS_EV Position */
+#define MXC_F_UART_INTEN_CTS_EV                        ((uint32_t)(0x1UL << MXC_F_UART_INTEN_CTS_EV_POS)) /**< INTEN_CTS_EV Mask */
+
+#define MXC_F_UART_INTEN_RX_OV_POS                     3 /**< INTEN_RX_OV Position */
+#define MXC_F_UART_INTEN_RX_OV                         ((uint32_t)(0x1UL << MXC_F_UART_INTEN_RX_OV_POS)) /**< INTEN_RX_OV Mask */
+
+#define MXC_F_UART_INTEN_RX_THD_POS                    4 /**< INTEN_RX_THD Position */
+#define MXC_F_UART_INTEN_RX_THD                        ((uint32_t)(0x1UL << MXC_F_UART_INTEN_RX_THD_POS)) /**< INTEN_RX_THD Mask */
+
+#define MXC_F_UART_INTEN_TX_OB_POS                     5 /**< INTEN_TX_OB Position */
+#define MXC_F_UART_INTEN_TX_OB                         ((uint32_t)(0x1UL << MXC_F_UART_INTEN_TX_OB_POS)) /**< INTEN_TX_OB Mask */
+
+#define MXC_F_UART_INTEN_TX_THD_POS                    6 /**< INTEN_TX_THD Position */
+#define MXC_F_UART_INTEN_TX_THD                        ((uint32_t)(0x1UL << MXC_F_UART_INTEN_TX_THD_POS)) /**< INTEN_TX_THD Mask */
```

Comment:
I think this should still be `INTEN_TX_HE` as on the max32655?

![image](https://github.com/analogdevicesinc/msdk/assets/147884137/342ed095-28db-4d66-a399-d07cffe09c1b)